web/server/codechecker_server/api/report_server.py

```diff
@@ -2125,10 +1996,7 @@ def getRunHistoryTagCounts(self, run_ids, report_filter, cmp_data, limit,
                 .outerjoin(report_cnt_q,
                            report_cnt_q.c.run_id == RunHistory.run_id) \
                 .filter(RunHistory.version_tag.isnot(None)) \
-                .filter(and_(report_cnt_q.c.detected_at <= RunHistory.time,
-                             or_(report_cnt_q.c.fixed_at.is_(None),
-                                 report_cnt_q.c.fixed_at >=
-                                 RunHistory.time))) \
+                .filter(get_outstanding_reports_filter(report_cnt_q.c)) \
```

Comment:
open reports filter? Shouldn't the query from the other PR be used when that is merged?